llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -5036,20 +5036,17 @@ static inline SDValue getPTrue(SelectionDAG &DAG, SDLoc DL, EVT VT,
 }
 
 static SDValue optimizeWhile(SDValue Op, SelectionDAG &DAG, bool IsSigned,
```

Comment:
minor nit: is it worth renaming this to `optimizeIncrementingWhile` to distinguish it further?